src/libraries/Common/src/System/Security/Cryptography/SlhDsa.cs

```diff
@@ -630,8 +743,12 @@ public string ExportEncryptedPkcs8PrivateKeyPem(
         ///   The number of bytes written to <paramref name="destination"/>.
         /// </returns>
         /// <exception cref="ArgumentException">
-        ///   <paramref name="destination"/> is too small to hold the public key.
+        ///   <paramref name="destination"/> is the incorrect length to receive the public key.
```

Comment:
"too small" is still correct as of this change, even if we're planning on changing it soon.